get_faiss_indexes.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
Can you please move this file to `src/main/python/parquet` as to not pollute the top-level structure?

Reply:
done